src/adt/set.c

```diff
@@ -74,7 +74,7 @@ static int
 set_cmp(const void *a, const void *b)
 {
 
-	return (a > b) - (a < b);
+	return (a < b) - (a > b);
```

Comment:
We didn't regenerate lexers after this change, so we didn't see. But it changed the ordering of states. I don't think it's a big deal, but it was making it annoying for me to test on a machine that didn't have sid installed.